test/common.sh

```diff
@@ -97,3 +108,25 @@ function test_setup() {
   echo ">> Uploading test images..."
   ${REPO_ROOT_DIR}/test/upload-test-images.sh || return 1
 }
+
+# Retrieve latest version from given Knative repository tags
+# On 'main' branch the latest released version is returned
+# On 'release-x.y' branch the latest patch version for 'x.y.*' is returned
+# Similar to hack/library.sh get_latest_knative_yaml_source()
+function get_latest_release_version() {
+    local repo_name="$1"
+    local major_minor=""
+    if is_release_branch; then
+      local branch_name
+      branch_name="$(current_branch)"
+      major_minor="${branch_name##release-}"
+    fi
+    local version
+    version="$(git ls-remote --tags --ref https://github.com/knative/"${repo_name}".git \
+      | grep "${major_minor}" \
+      | cut -d '-' -f2 \
+      | cut -d 'v' -f2 \
+      | sort -Vr \
+      | head -n 1)"
+    echo "${version}"
```

Comment:
you could just skip the version variable by directly running the git-pipeline without executing it in a subshell

Reply:
Well `shellcheck` complained about masking return value and legacy backthick.

Reply:
yeah, was just a cosmetic comment that you don't need backticks at all, just call git without a variable (it will echo the result anyway). but if this works better here, then I'm fine.